toolkit/tools/internal/pkggraph/pkggraph.go

```diff
@@ -620,6 +629,17 @@ func (g *PkgGraph) AllNodesFrom(rootNode *PkgNode) []*PkgNode {
 
 // AllRunNodes returns a list of all run nodes in the graph
 func (g *PkgGraph) AllRunNodes() []*PkgNode {
+	nodes := make([]*PkgNode, 0, g.Nodes().Len())
+	for _, n := range g.AllNodes() {
+		if n.Type == TypeLocalRun || n.Type == TypeRemoteRun {
```

Comment:
@PawelWMS I'm trying to recall, do we consider prebuilt nodes "runnable"? Probably not something to worry about right now, but maybe we want to do a pass and sanity check how we handle them.

Reply:
You are right. PreBuiltNode can be considered as a RunNode. But as no tool has any action upon the PreBuiltNodes. So, it is not being returned from any function.

Reply:
As far as I can recall, @0xba1a is correct: they can be considered a run-time dependency, but they are never retrieved and saved anywhere (except for existing as a node in the graph that is). And think that's an inconsistency we should fix, but that's definitely out of scope for this PR. I've added that, I must clean-up. ;)